04-suspense-and-async-mode/README.md

```diff
@@ -0,0 +1,237 @@
+# React Suspense and Concurrent Mode
+
+## Code Splitting
+
+* Bundling your application is great, but as your app grows, your bundle size will grow too, especially if you include large 3rd party libraries
+* This may have a huge impact on the time to first load and UX, especially on slow networks
+* The bundle can be split into smaller chunks where the most important ones can be loaded first and then every other secondary are lazily loaded - **code splitting**
+
+### What Is Code Splitting
+
+* Multiple bundles are created that are loaded dynamically, only at the time they are needed
+* Can significantly improve performance of your app
+
+### React.lazy
+
+* lazy() is a function that enables you to to render a dynamic import as a regular component
```

Comment:
typo: `to to`

Reply:
Good catch, thank you, fixed in de4a08fd9d201603c558b435bfad1d09ae1cd92f